api/src/main/resources/workbench.yaml

```diff
@@ -1256,7 +1256,7 @@ paths:
           name: copyNotebookRequest
           required: true
           schema:
-            $ref: '#/definitions/CopyNotebookRequest'
+            $ref: '#/definitions/CopyRequest'
```

Comment:
should the `name` attribute above at line 1256 be changed as well?

Reply:
Unsure. I actually am happy with it as is, because it is still a request to copyNotebook? Do you have a preference?